node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,8 @@
+import { writeFile } from 'node:fs/promises';
+const content = Math.random() * 100;
+const newFile = writeFile('random.txt', content.toString());
+try {
+  await newFile;
```

Comment:
Same here. This works since `newFile` is a promise, but it's unusual. People will find your code easier to read and work with if you `await` function calls.